packages/katello/katello/service-wait

```diff
@@ -57,46 +53,21 @@ before_start() {
 # after start or restart
 after_start() {
   case "$SERVICE" in
-    tomcat|tomcat6)
-      # RHBZ 789288 - wait until data port is available
-      wait_for_url $TOMCAT_TEST_URL
-      ;;
-    foreman)
-      wait_for_url $FOREMAN_TEST_URL
-      ;;
-    katello)
-      wait_for_url $KATELLO_TEST_URL
-      ;;
-    mongod)
-      # RHBZ 824405 - wait until service is avaiable
-      for i in {1..$WAIT_MAX}; do mongo --eval "printjson(db.getCollectionNames())" 2>/dev/null 1>&2 && break; sleep 1; done
-      sleep $ADDITIONAL_SLEEP
-      ;;
     postgresql)
-      # RHBZ 800534 for RHEL 6.x - pg sysvinit script return non-zero when PID is not created in 2 seconds
-      RETVAL=0
-      # and wait until port is really avaiable
       for i in {1..$WAIT_MAX}; do ss -ln | grep -q ":$POSTGRES_PORT\s" && break; sleep 1; done
       # and create lock and pid files if they does not exist
```

Comment:
I think this could be removed. I'd trust the init scripts to do this right and many systemd scripts no longer care about these files so they might be obsolete anyway.